src/components/Comment/index.tsx

```diff
@@ -441,26 +503,29 @@ export default class CommentComponent extends React.Component<CommentProps> {
 
     return (
       <>
-        <p className="comment__text">{comment.text}</p>
-        {this.renderAuthorDate()}
-        {this.renderReplies({ hideNewReply: true })}
-        <div className="comment__error">
-          {strings.DELETE_ERROR}
-          <button
-            type="button"
-            className="comment__button"
-            onClick={onClickCancel}
-          >
-            {strings.CANCEL}
-          </button>
-          <button
-            type="button"
-            className="comment__button"
-            onClick={onClickRetry}
-          >
-            {strings.RETRY}
-          </button>
+        <div className="comment__container">
+          {this.renderMessage()}
+          <p className="comment__text">{comment.text}</p>
```

Comment:
This comment container markup is repeated 6 times in the file, and the file is 600+ lines long. I think this is overdue for more break-up into smaller components.